components/dpu-workflow/src/batch_pdf_classifier.py

```diff
@@ -16,12 +16,14 @@
 import re
 from typing import Optional
 
+from google.api_core.gapic_v1.client_info import ClientInfo  # type: ignore
 from google.api_core.client_options import ClientOptions # type: ignore # pylint: disable = no-name-in-module # pylint: disable = import-error
 from google.api_core.exceptions import InternalServerError # type: ignore # pylint: disable = no-name-in-module # pylint: disable = import-error
 from google.api_core.exceptions import RetryError # type: ignore # pylint: disable = no-name-in-module # pylint: disable = import-error
 from google.cloud import documentai  # type: ignore # pylint: disable = no-name-in-module # pylint: disable = import-error
 from google.cloud import storage # type: ignore # pylint: disable = no-name-in-module # pylint: disable = import-error
 
+USER_AGENT = "cloud-solutions/eks-docai-v1"
```

Comment:
Side note - in v1.2, this `batch_pdf_classifier.py` is NOT being used, but this change is fine nonetheless.

Reply:
Yes, your are right.